webapp/channels/src/components/commercial_support_modal/commercial_support_modal.tsx

```diff
@@ -145,29 +145,43 @@ export default class CommercialSupportModal extends React.PureComponent<Props, S
                 </Modal.Header>
                 <Modal.Body>
                     <div className='CommercialSupportModal'>
-                        <FormattedMarkdownMessage
+                        <FormattedMessage
                             id='commercial_support.description'
-                            defaultMessage={'If you\'re experiencing issues, [submit a support ticket](!{supportLink}). To help with troubleshooting, it\'s recommended to download the Support Packet below that includes more details about your Mattermost environment.'}
+                            defaultMessage={'If you\'re experiencing issues, <a>submit a support ticket</a>. To help with troubleshooting, it\'s recommended to download the Support Packet below that includes more details about your Mattermost environment.'}
                             values={{
-                                supportLink,
+                                a: (chunks: string) => (
+                                    <ExternalLink
+                                        href={supportLink}
+                                        //TODO: check what location is and where it should
+                                        location='supportLink'
+                                    >
+                                        {chunks}
+                                    </ExternalLink>
+                                ),
                             }}
                         />
                         {showBannerWarning &&
                             <AlertBanner
                                 mode='info'
                                 message={
-                                    <FormattedMarkdownMessage
+                                    <FormattedMessage
                                         id='commercial_support.warning.banner'
-                                        defaultMessage='Before downloading the Support Packet, set **Output Logs to File** to **true** and set **File Log Level** to **DEBUG** [here](!/admin_console/environment/logging).'
+                                        defaultMessage='Before downloading the Support Packet, set <strong>Output Logs to File</strong> to <strong>true</strong> and set <strong>File Log Level</strong> to <strong>DEBUG</strong> [here](!/admin_console/environment/logging).'
+                                        values={{
+                                            strong: (chunks: string) => <strong>{chunks}</strong>,
+                                        }}
                                     />
                                 }
                                 onDismiss={this.hideBannerWarning}
                             />
                         }
                         <div className='CommercialSupportModal__packet_contents_download'>
```

Comment:
since this whole sentence we can make this div to be strong instead. and message could be just `Select your Support Packet contents to download`